src/js/views/clinicians/clinicians-all_views.js

```diff
@@ -116,7 +129,12 @@ const LayoutView = View.extend({
   className: 'flex-region',
   template: hbs`
     <div class="list-page__header">
-      <div class="list-page__title"><span class="list-page__title-icon">{{far "users-cog"}}</span>{{ @intl.clinicians.cliniciansAllViews.layoutView.title }}</div>
+      <div class="flex list-page__title">
+        <div>
```

Comment:
This empty `<div>` helps align the title icon and title text elements.

Without the empty `<div>`:

![Screen Shot 2022-05-06 at 11 19 53 AM](https://user-images.githubusercontent.com/35355575/167177368-7ed93233-e58a-4003-9300-c323f3ccd79b.png)

With the `<div>` added:

![Screen Shot 2022-05-06 at 11 20 18 AM](https://user-images.githubusercontent.com/35355575/167177389-f7c61c41-544a-4f2b-9f2a-2a79586be138.png)